### PR TITLE
build Homepage

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Screenshot (166)
Screenshot (167)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (hiteshverma27) [inline public/index.html]: Change this to title

File: src/components/navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  };
+  const closeMenu = () => {
+    setIsMenuShow(false);
+  };
+  return (
+    <nav className="navbar semibold-font-weight">
+      <div className="openMenu">
+        <i className="fa fa-bars" onClick={showMenu}></i>
+      </div>
+      <a className="nav-icon-link nav-link link-no-style hide-icon" href="">
```
Comment (hiteshverma27) [inline src/components/navbar/Navbar.js:17]: use Link tag

File: src/components/footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./footer.css";
+
+export function Footer() {
+  return (
+    <div>
+      <footer className="footer">
+        <div className="footer-items">
+          <div className="made-by">
+            Made with <i class="fas fa-heart footer-icon"></i> By Ishita
```
Comment (hiteshverma27) [inline src/components/footer/Footer.js:10]: if possible add a more descriptive footer with more links

File: src/components/category/Category.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import "./category.css";
+
+
+export function Category() {
+  return (
+    <div className="categories">
+      <div className="category-title">Categories</div>
+      <div className="category-list">
+        {categories.map((category) => (
+          <div className="card">
```
Comment (hiteshverma27) [inline src/components/category/Category.js]: You can make a component card and return it with props.

**Review by hiteshverma27 — APPROVED** [review]
Left few suggestions
- Good folder structure 
- Clean code 
- Readable code

File: src/components/category/Category.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { categories } from "../../backend/db/categories";
```
Comment (anupkgurung) [inline src/components/category/Category.js:2]: use api call for instead of importing db component

**Review by anupkgurung — APPROVED** [review]
Try to keep the merge clean. 
Use API call instead of importing db.
use Link instead of anchor as the project is already CRA.

File: public/index.html (hunk 1 above)
Comment (ishitakeshawani) [reply public/index.html]: done

File: src/components/navbar/Navbar.js (hunk 2 above)
Comment (ishitakeshawani) [reply src/components/navbar/Navbar.js:17]: added routes, link in the next PR. So not changing here.

File: src/components/footer/Footer.js (hunk 3 above)
Comment (ishitakeshawani) [reply src/components/footer/Footer.js:10]: will add after p0 features

File: src/components/category/Category.js (hunk 5 above)
Comment (ishitakeshawani) [reply src/components/category/Category.js:2]: Used it in the next PR. So not changing here.

File: src/components/category/Category.js (hunk 4 above)
Comment (ishitakeshawani) [reply src/components/category/Category.js]: done

**Pull request merged**